packages/card/index.js

```diff
@@ -0,0 +1,116 @@
+import { FabricWebComponent } from '../utils';
+import { card as c } from '@fabric-ds/component-classes';
+import { classNames } from '@chbphone55/classnames';
+
+class FabricCard extends FabricWebComponent {
+  static get observedAttributes() {
+    return ['onlick', 'selected'];
+  }
+
+  attributeChangedCallback(name, _, newValue) {
+    switch (name) {
+      case 'onclick':
+        this[name] = new Function('return ' + newValue)();
+        break;
+      case 'selected':
+        this[name] = newValue == 'false' ? false : true;
+        break;
+      default:
+        this[name] = newValue;
+    }
+
+    this.id =
+      this.id ||
```

Comment:
Do we need to set an id internally?

Reply:
For re-rendering purposes, yeah. I tried without with no success.